src/main/java/com/targomo/client/api/pojo/Geometry.java

```diff
@@ -56,4 +56,9 @@ public String getData() {
     public Integer getCrs() {
         return crs;
     }
+
+    @Override
+    public int hashCode() {
+        return data.hashCode() * type.hashCode() * crs * 31;
```

Comment:
please next time use `Objects.hash(data, type, crs)` or the lombok annotation - otherwise you now will potentially get NPEs

Reply:
Updated and created a new pull request https://github.com/targomo/targomo-java/pull/71